infrastructure-playbooks/client-external-cluster.yml

```diff
@@ -0,0 +1,32 @@
+- name: deploy a client node for an external cluster
+
+  vars:
+    _ceph_conf_key_directory: "{{ ceph_conf_key_directory | default('/etc/ceph') }}"
+    _cluster: "{{ cluster | default('ceph') }}"
+
+  hosts:
+    - "{{ client_group_name | default('clients') }}"
+    - "{{ mon_group_name | default('mons') }}"
```

Comment:
I think we need to do a `add_host` instead, we can't expect a monitor section.
I think we are are going to get an IP and then we need to add it as a new host on the fly during the play.